services/backend/src/database/schema.prisma

```diff
@@ -521,7 +568,7 @@ model User {
   educations           Education[]
   experiences          Experience[]
   relocationLocations  RelocationLocation[]
-  connections          User[]
+  connections          User[]                @relation("UserToUser")
   user                 User?                 @relation("UserToUser", fields: [userId])
```

Comment:
Can you have relation names be identical as another one?

Reply:
what do you mean? for the UserToUser relation?

Reply:
Yes, nevermind I think I understand, but why is this being added in this PR, is it just the vscode formatter that added this?

Reply:
Yeah, prettier automatically added it. Not sure why now maybe after an update of one of the packages.